.aws/config

```diff
@@ -0,0 +1,4 @@
+[profile ci-user]
+role_arn = arn:aws:iam::248295656887:role/ci-user
```

Comment:
Should we be exposing `248295656887` here? This is a public repo

Reply:
good point, let me see if there's some way to remove it from the repo